### ISSUE TITLE
Converting the gh-pages branch content to jekyll

### ISSUE THREAD

**Issue by RubenOlsen** [svandecappelle/jQPlot#45]
# Problem description

Using Github Pages for the documentation makes better sense that the wiki for several reasons, but not limited to:

* Easier to maintain on a technical level. The tool-support is better handling files, than using a web-brower to update a wiki.
* Having the possibility to generate API docs, rather than manually updating information. Manual updates will never be "up-to-date".
* We need a good alternative to the old jQplot web-pages - Github Pages are the answer.
* A well structured website it easier to navigate than a wiki. Wikis have their place with regards to documentation - but maybe not for the kind of documentation we need which is API and interactive examples. 

# Todo

Convert the current gh-pages to jekyll.

# Done criteria

When the current gh-pages are jekyll-ified without changing the current look and feel

# Things to keep in mind

We must document in the CONTRIBUTING.md file (#44) what kind of documentation goes where.

**Comment by johanbove** [comment]
Agreed! So until @svandecappelle makes you a contributor, I suggest starting to build the Jekyll docs in a fork.

**Comment by svandecappelle** [comment]
Added to contributor ;) Thanks for supporting the renew of jqplot ;)

**Comment by RubenOlsen** [comment]
I have given the question with regards to when to use the wiki and when to use the GH pages for documentation.

My suggestion is as follows: How about we use wiki-pages for documenting everything that has to do with the development of jQPlot, while the GH pages deals with everything that has to do with how to use jQPlot in your own project.

If this makes sense, this information should be present in the CONTRIBUTING.md  (see #44).

**Comment by svandecappelle** [comment]
It seems that broken CSS style too =)

Fonts of header h2.plain
Images

**Comment by RubenOlsen** [comment]
Now we are talking about taste.

My reasons for using class="plain" inside the documentation is that a swigly font is not always appropriate. Documentation should be easy to read, a swigly headings is not easy to read.

I have created #52 to fix some CSS issues - I'll see what I'm going to do with the headers. I think we should keep the swigly fonts on the front page and on the main documentation pages - but use a plain font for the headers inside the documentation pages.

**Comment by svandecappelle** [comment]
```I think we should keep the swigly fonts on the front page and on the main documentation pages - but use a plain font for the headers inside the documentation pages.```

@RubenOlsen: Agree with that
